src/ws/test-remotectlcertificate.c

```diff
@@ -245,12 +289,25 @@ static const TestFixture fixture_preinstall_combined_key_last = {
     .files = no_files,
 };
 
+static void
+detect_openssl (void)
+{
+  gint exit_status;
+  g_autofree gchar *stdout = NULL;
+  g_autofree gchar *stderr = NULL;
+
+  g_assert (g_spawn_command_line_sync ("which openssl", &stdout, &stderr, &exit_status, NULL));
```

Comment:
`g_find_program_in_path()`

Reply:
:heart_eyes: Nice! I had a gut feeling that glib had such a thing, but I couldn't find it any more. Changed.